### PR TITLE
Added cppcheck to base images for circleci

### PR DESCRIPTION
**Pull request by CryptoNinjaGeek**
### Scope & Purpose

Install cppcheck in our base circleci images

### CODE REVIEW

File: scripts/docker/build/arangodb-build.Dockerfile [hunk 1]
```diff
@@ -13,6 +13,9 @@ COPY install-openldap.sh /tools/
 RUN [ "/tools/install-openldap.sh", "3.1.2" ]
 RUN apk --no-cache del perl
 
+COPY install-cppcheck.sh /tools/
+RUN [ "/tools/install-cppcheck.sh", "2.11" , "2.11.0" ]
```
Comment (KVS85) [inline scripts/docker/build/arangodb-build.Dockerfile:17]:
```
COPY install-cppcheck.sh /tools/
RUN if [ "$(apk --print-arch)" = "x86_64" ] ; then /tools/install-cppcheck.sh 2.11 2.11.0 ; fi
```

**Review by KVS85 — CHANGES_REQUESTED** [review]
We probably don't need cppcheck in ARM build image.

File: scripts/docker/build/arangodb-build.Dockerfile (hunk 1 above)
Comment (CryptoNinjaGeek) [reply scripts/docker/build/arangodb-build.Dockerfile]: @KVS85 Ok, I removed it from arm images
The reason why I didn't do a check before, was to have a consistent set of tools on both architectures

**Review by KVS85 — APPROVED** [review]

**Comment by KVS85** [comment]
https://hub.docker.com/layers/arangodb/build-alpine/3.16-gcc11.2-openssl3.1.2-e5e06cdaa21/images/sha256-51c78b6223111bbe7919d634df77439e1daf8a1fbca7ff95509a0d25f2f3428f

**Pull request merged**